CommonLibSF/include/SFSE/Utilities.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "REL/Relocation.h"
+#include "SFSE/API.h"
+
+namespace SFSE
+{
+	namespace stl
+	{
+		template <class T, unsigned TSize = 5>
+		constexpr void write_thunk_call() noexcept
```

Comment:
- Prefer using std::size_t for these
- Template parameter naming isn't consistent with CLIB (Size vs TSize)